### ISSUE TITLE
Volume list view creates volume

### ISSUE THREAD

**Issue by egeldenhuys** [TripleParity/docks-ui#101]
Payload
```
{"Name":"tardis2","Labels":{"com.example.some-label":"some-value","com.example.some-other-label":"some-other-value"},"Driver":"local"}
```

**Comment by CDuPlooy** [comment]
'payload' 😆 

In some of the components you'll find code creating something to display because either the ui component has not been finished and/or I was playing and forgot to remove that line. In this case we were waiting on the creation of a ui component.

**Comment by egeldenhuys** [comment]
Fixed by #147
